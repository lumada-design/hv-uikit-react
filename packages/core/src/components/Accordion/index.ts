export { default as accordionClasses } from "./accordionClasses";
export * from "./accordionClasses";
export * from "./Accordion";
