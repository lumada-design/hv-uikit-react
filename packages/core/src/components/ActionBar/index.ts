export { default as actionBarClasses } from "./actionBarClasses";
export * from "./actionBarClasses";
export * from "./ActionBar";
