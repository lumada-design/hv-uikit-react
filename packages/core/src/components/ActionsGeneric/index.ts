export { default as actionsGenericClasses } from "./actionsGenericClasses";
export * from "./actionsGenericClasses";
export * from "./ActionsGeneric";
