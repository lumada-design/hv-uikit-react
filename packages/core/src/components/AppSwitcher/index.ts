export * from "./AppSwitcher";
export { HvAction as HvAppSwitcherAction } from "./Action";
