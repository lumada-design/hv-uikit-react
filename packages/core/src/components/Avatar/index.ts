export { default as avatarClasses } from "./avatarClasses";
export * from "./avatarClasses";
export * from "./Avatar";
