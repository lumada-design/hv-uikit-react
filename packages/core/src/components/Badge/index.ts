export { default as badgeClasses } from "./badgeClasses";
export * from "./badgeClasses";
export * from "./Badge";
