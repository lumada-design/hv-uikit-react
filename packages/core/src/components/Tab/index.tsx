export { default as tabClasses } from "./tabClasses";
export * from "./tabClasses";
export * from "./Tab";
