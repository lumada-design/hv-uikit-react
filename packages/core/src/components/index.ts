export * from "./Box";
export { Box as HvBox } from "./Box";
export * from "./Button";
export { Button as HvButton } from "./Button";
export * from "./Dropdown";
export { Dropdown as HvDropdown } from "./Dropdown";
export * from "./Header";
export {
  Actions as HvHeaderActions,
  Brand as HvHeaderBrand,
  Header as HvHeader,
  Navigation as HvHeaderNavigation,
} from "./Header";
export * from "./Typography";
export { Typography as HvTypography } from "./Typography";
