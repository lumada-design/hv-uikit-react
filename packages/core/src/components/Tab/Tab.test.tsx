import { render, screen } from "@testing-library/react";
import { describe, expect, it } from "vitest";
import { HvTab } from "@core/components";

describe("Tab", () => {
  it("should render correctly", () => {
    render(<HvTab label="Clickable tab" />);

    expect(
      screen.getByRole("tab", { name: "Clickable tab" })
    ).toBeInTheDocument();
  });
});
