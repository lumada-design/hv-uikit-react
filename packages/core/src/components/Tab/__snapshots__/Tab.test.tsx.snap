// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Tab > should render correctly 1`] = `
<div>
  <button
    class="MuiButtonBase-root MuiTab-root HvTab-root MuiTab-textColorInherit css-mxbnpo-StyledTab evi6req0 css-cjctlb-MuiButtonBase-root-MuiTab-root"
    role="tab"
    tabindex="-1"
    type="button"
  >
    Clickable tab
  </button>
</div>
`;
